packages/friends/yml/en/elizabeth-ussher.yml

```diff
@@ -11,11 +33,37 @@ documents:
       Waterford.
     slug: letters
     filename: Ussher_Letters
-    description: TODO
+    description:
+      After a union of seventeen years with her husband, Elizabeth Ussher (1749-1817) was
```

Comment:
This description is just a hair different from the one above, by like 2 words. Did you mean for that to be the case? Or maybe did you alter one of them after you copy/pasted and forgot to alter the other?